sdk/src/main/scala/com/horizen/api/http/SidechainWalletApiRoute.scala

```diff
@@ -134,6 +138,121 @@ case class SidechainWalletApiRoute(override val settings: RESTApiSettings,
     }
   }
 
+  /**
+   * Import a private key inside the wallet
+   */
+  def importSecret: Route = (post & path("importSecret")) {
+    withAuth {
+      entity(as[ReqImportSecret]) { body =>
+        val secret = sidechainSecretsCompanion.parseBytes(BytesUtils.fromHexString(body.privKey))
+        val future = sidechainNodeViewHolderRef ? LocallyGeneratedSecret(secret)
+        Await.result(future, timeout.duration).asInstanceOf[Try[Unit]] match {
+          case Success(_) =>
+            ApiResponseUtil.toResponse(RespCreatePrivateKey25519(secret.publicImage()))
```

Comment:
RespCreatePrivateKey25519 name seems outdated.
Probably we should name it just `RespCreatePrivateKey`

Reply:
Done